src/store/store-items.js

```diff
@@ -178,11 +217,38 @@ const getters = {
     }, [])
     return itemsCart
   },
+  priceTotal: state => {
+    return itemsPriceTotal(state.items)
+  },
+  itemsTotal: state => {
+    return itemsNumber(state.items)
+  },
+  cartTotal: (state, getters) => {
+    const itemsCart = getters.itemsCart
+    return itemsNumber(itemsCart)
+  },
   getItemById: (state) => (id) => {
     return state.items[id]
   }
 }
 
+const itemsPriceTotal = items => {
+  const reducerSum = (sum, i) => sum + totalPrice(i)
+  const itemsValues = Object.values(items)
+  return Math.round(itemsValues.reduce(reducerSum, 0) * 100) / 100
+}
+
+const totalPrice = item => {
+  return item.price * item.quantity
+}
+
+const itemsNumber = items => {
+  if (!items) {
+    return 0
+  }
+  return Object.keys(items).length
+}
```

Comment:
Above helpers and getter could be used directly in the list instead of calculate them in the UI component. Possible Clean up